projects/laji/src/app/+taxonomy/taxon/info-card/taxon-identification/taxon-identification.component.ts

```diff
@@ -75,7 +164,11 @@ export class TaxonIdentificationComponent implements OnChanges, AfterViewInit, O
 
     this.subscription.add(
       this.children$.subscribe((t) => {
-        this.children = t;
+        this.children = t.map(child => {
```

Comment:
```
descriptionSources: Array<string> = [];
  speciesCardAuthors: Array<string> = [];
  speciesCardAuthorsTitle: string | undefined = undefined;
```
All of these variables are pretty weird, because they are directly related to the same data query, but are listed as separate variables. This extra state & side effects adds some mental overhead that's unnecessary. A better solution would be to tie all of them together, eg. replace the `children` class variable with something like:
```
interface Data {
  children;
  descriptionSources;
  speciesCardAuthors;
  speciesCardAuthorsTitle;
}
```
and get rid of the descriptionSources etc. class variables.